sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/ReactiveCosmosTemplate.java

```diff
@@ -64,7 +66,7 @@
      */
     public ReactiveCosmosTemplate(CosmosAsyncClient client, String databaseName,
                                   CosmosConfig cosmosConfig, MappingCosmosConverter mappingCosmosConverter) {
-        this(new CosmosFactory(client, databaseName), cosmosConfig, mappingCosmosConverter);
+        this(new CosmosFactory(client, databaseName), cosmosConfig, mappingCosmosConverter, null);
```

Comment:
Same here - support for multi-database configuration. Auditing should work in that case too.

Reply:
added a new constructor that includes IsNewAwareAuditingHandler as a parameter for use in multi-database configuration. Updated sample in DatabaseConfiguration.java to use this new constructor with auditing enabled.